robocorp-code/vscode-client/src/activities.ts

```diff
@@ -64,6 +64,34 @@ export async function cloudLogin(): Promise<boolean> {
     return true;
 }
 
+export async function cloudLogout(): Promise<void> {
+    console.log('Cloud Logout() called.')
+    let loggedOut: boolean;
+    // Start this in parallel while we ask the user for info.
+    let isLoginNeededPromise: Thenable<ActionResult> = commands.executeCommand(
```

Comment:
I think you could just do:
```
let isLoginNeeded: ActionResult = commands.executeCommand(roboCommands.ROBOCORP_IS_LOGIN_NEEDED_INTERNAL);
```

To avoid having to assign to the `Thenable`.

Also, the processing isn't really being done in parallel in this case (the comment is a bit misleading here as it asks and then awaits just in the next line, so, the comment should be removed too).

Reply:
Fixes to client command logic